templates/include/address-space-definitions.yaml

```diff
@@ -1249,6 +1546,22 @@ data:
             - name: broker-templates
               configMap:
                 name: broker-templates
+    - apiVersion: v1
+      kind: Service
+      metadata:
+        annotations:
+          addressSpace: ${ADDRESS_SPACE}
+        labels:
+          app: enmasse
+        name: qdrouterd-headless
+      spec:
+        clusterIP: None
+        ports:
+        - name: inter-router
+          port: 55672
+          targetPort: 55672
```

Comment:
Can/should these ports be removed from the «messaging» service now?

Reply:
They could be. That seems an orthogonal question to this change though in that I don't believe they are used before or after this patch.